etc/config/c++.amazon.properties

```diff
@@ -4184,6 +4184,13 @@ libs.taojson.versions=trunk
 libs.taojson.versions.trunk.version=trunk
 libs.taojson.versions.trunk.path=/opt/compiler-explorer/libs/taojson/trunk/include
 
+libs.taskflow.name=taskflow
+libs.taskflow.description=taskflow is a C++ Header-only General-purpose Parallel and Heterogeneous Task Programming System
+libs.taskflow.url=https://github.com/taskflow/taskflow
+libs.taskflow.versions=trunk
+libs.taskflow.versions.trunk.version=trunk
+libs.taskflow.versions.trunk.path=/opt/compiler-explorer/libs/taskflow/trunk/include
```

Comment:
Unfortunately this path doesn't exist.

```
$ ls /opt/compiler-explorer/libs/taskflow/trunk/
3rd-party/  benchmarks/  cmake/  CMakeLists.txt  docs/  doxygen/  examples/  image/  LICENSE  README.md  sandbox/  taskflow/  TaskflowConfig.cmake.in  tfprof/  unittests/
```
these are the directories available after installation. It's not clear to me which of these should be used as the include directory.

Reply:
Maybe `/opt/compiler-explorer/libs/taskflow/trunk/` itself? that would mean an include of `taskflow/taskflow.hpp` would work, maybe?

Reply:
Thanks for your suggestions, and I just remove the include directory at the end. Let me know if that works.